src/app/containers/AudioVideo/index.jsx

```diff
@@ -51,16 +56,27 @@ const AudioVideoContainer = ({ blocks }) => {
   let ParentWrapper = NestedGridParentLarge;
   let ChildWrapper = NestedGridItemChildLarge;
   let Container = GridItemConstrainedLargeNoMargin;
-  let width = '100%';
-  let height = '100%';
+  let widthAndHeight = `
+    height: 100%;
+    width: 100%;
+  `;
 
   if (orientation === 'Portrait') {
     ParentWrapper = NestedGridParentSmall;
     ChildWrapper = NestedGridItemChildSmall;
     Container = GridItemConstrainedSmall;
     wrapperSpan.default = '4';
-    width = '100%';
-    height = `853px`;
+
+    widthAndHeight = `
+      @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+          height: calc(((100vw - 2rem) * 3/5 ) * 16/9);
+          width: 100%;
+      }
+      @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+        height: calc(((100vw - (100vw - 1280px) + 2rem) * 8/20 ) * 16/9);
```

Comment:
I've added `(100vw - 1280px)` because at the 1280+ breakpoint the width of the page increases in the margins only